rocketpy/control/controller.py

```diff
@@ -0,0 +1,132 @@
+from ..prints.controller_prints import _ControllerPrints
+
+
+class _Controller:
+    """A class for storing and running controllers on a rocket. Controllers
+    have a controller function that is called at a specified sampling rate
+    during the simulation. The controller function can access and modify
+    the objects that are passed to it. The controller function also stores the
+    variables of interest in the objects that are passed to it."""
+
+    def __init__(
+        self,
+        interactable_objects,
```

Comment:
Altough the word 'interactable' makes a lot of sense, it doesn't exist in the most commom English dictionaries. 

https://www.oxfordlearnersdictionaries.com/us/spellcheck/english/?q=interactable

I believe the word you're looking for is "interactive".

I could simply search and replace it but in this case I prefer open this comment to see if you have any different opinion.

Reply:
Wow, good catch

Ill change it asap

Reply:
Solved here: [bf07e2e](https://github.com/RocketPy-Team/RocketPy/pull/426/commits/bf07e2e260a5b8352fceda328a7ae32528e76f5a)

Thanks!